### ISSUE TITLE
A required class was missing while executing com.microsoft.azure:azure-functions-maven-plugin:1.0.0-beta-2:package: javax/xml/bind/JAXBException

### ISSUE THREAD

**Issue by jdneo** [Microsoft/azure-maven-plugins#162]
This issue occurs when user are using JDK 9+.

The root cause is that:

The JAXB APIs are considered to be Java EE APIs, and therefore are no longer contained on the default class path in Java SE 9.

workaround is adding related dependencies in the pom file:

```xml
<dependency>
    <groupId>javax.xml.bind</groupId>
    <artifactId>jaxb-api</artifactId>
    <version>2.2.7</version>
</dependency>
<dependency>
    <groupId>com.sun.xml.bind</groupId>
    <artifactId>jaxb-core</artifactId>
    <version>2.2.7</version>
</dependency>
<dependency>
    <groupId>com.sun.xml.bind</groupId>
    <artifactId>jaxb-impl</artifactId>
    <version>2.2.7</version>
</dependency>
```

**Comment by jdneo** [comment]
May related with this issue: https://github.com/Microsoft/ApplicationInsights-Java/issues/674

**Comment by rhummelmose** [comment]
Are we going to fix this or? @jdneo

**Comment by thomasvdb** [comment]
Any update on this? Tried the workaround by adding the related dependencies but that didn't work.

**Comment by jdneo** [comment]
@all, We still hope that this issue could be addressed. But unfortunately, it's not on the roadmap of the AI-Java-SDK so far.

For anyone who meets this problem, please also leave your comments [here](https://github.com/Microsoft/ApplicationInsights-Java/issues/674) to highlight this issue.

Thanks.

**Comment by Flanker32** [comment]
Fixed in #610.
